src/content/docs/crowdin/project-management/sources/string-management.mdx

```diff
@@ -34,6 +34,57 @@ By default, all source strings of the project are displayed in **Sources > Strin
 
 <Image src={stringFilter} alt="Filter Strings" />
 
+## Adding Strings
+
+You can add strings to your projects directly in Crowdin.
+
+For file-based projects, this feature is available only for file formats that support [online editing](#formats-that-support-online-editing).
```

Comment:
Move it to the File-based project tab, after the steps component.

Reply:
Moved